pyproject.toml

```diff
@@ -309,7 +309,9 @@ sbx = [
     "roiextractors>=0.5.12",
 ]
 scanimage = [
-    "roiextractors>=0.5.12",
+    "roiextractors @ git+https://github.com/catalystneuro/roiextractors.git@main",
```

Comment:
This git dependency is causing the dev docker build to fail (see https://github.com/catalystneuro/neuroconv/actions/runs/14963716648/job/42029898852).

I'm not sure how to fix except to update after a new roiextractors release, but maybe we should avoid git dependencies moving forward.

Reply:
Yes, we should. This was an oversight of mine. 

Can we make a release of roiextractors to fix this?

Reply:
Yes, but do you think we can work through some of the pending PRs first?

Reply:
Yes, let me know in slack if you need help with any of them.